doc/api.yml

```diff
@@ -1272,6 +1394,37 @@ components:
       enum:
         - ENABLED
         - DISABLED
+    StudentMissing:
+      type: object
+      properties:
+        id:
+          type: string
+        pointing_datetime:
+          type: string
+          format: date-time
+        place:
+          type: string
+        is_late:
+          type: boolean
+        is_missing:
```

Comment:
Don't understand, the component name is StudentMissing, why do you add an attribute is_missing here ?

Reply:
Component name changed to StudentPointing